lib/core/src/lib/identity-user-info/identity-user-info.component.ts

```diff
@@ -54,11 +54,12 @@ export class IdentityUserInfoComponent implements OnDestroy {
     @Input()
     showName: boolean = true;
 
-    /** When the username is shown, this defines its position relative to the user info button.
+    /**
+     * When the username is shown, this defines its position relative to the user info button.
      * Can be `right` or `left`.
      */
     @Input()
-    namePosition: string = 'right';
+    namePosition: 'right' | 'left';
```

Comment:
Is not that breaking change? Is that expected? In previous version it has 'right' as default value, now it will be undefined I guess?

Reply:
good catch, fixed